app/uk/gov/hmrc/fhregistrationfrontend/views/forms/national_insurance_number.scala.html

```diff
@@ -41,40 +40,82 @@
   Messages("fh.national_insurance_number.confirm.label")
 }
 
-@main_template(title = titlePrefix + title, scriptElem = Some(pageScripts)) {
+@conditionalHtml = @{
+    viewHelpers.govukInput(Input(
+        id = nationalInsuranceNumberKey,
+        name = nationalInsuranceNumberKey,
+        label = Label(
+            isPageHeading = true,
+            content = HtmlContent(s"<strong>${messages("fh.national_insurance_number.label")}</strong>")
+        ),
+        spellcheck = Some(false),
+        classes = "govuk-!-width-one-half",
+        hint = Some(Hint(
+            content = Text(messages("fh.national_insurance_number.hintText"))
+        )),
+        errorMessage =
+            nationalInsuranceNumberForm.error(nationalInsuranceNumberKey).map{ e =>
+                ErrorMessage(
+                    content = Text(messages(s"fh.${e.key}.${e.message}")),
+                    visuallyHiddenText = Some(messages("generic.errorPrefix"))
+                )
 
-  <link rel="stylesheet" href='@routes.Assets.at("stylesheets/fh.css")'/>
+            },
+        value = nationalInsuranceNumberForm(nationalInsuranceNumberKey).value
+    ))
+}
+
+@layout(title = titlePrefix + title) {
```

Comment:
include the pagescript, as you mentioned earlier they are needed. I will add them in my Pr where it's not there before rebasing

Reply:
placed back in.  Thank you - I pulled it out because of syntax but forgot to re-add it.